keyboards/keycapsss/plaid_pad/info.json

```diff
@@ -0,0 +1,31 @@
+{
+    "keyboard_name": "Plaid-Pad",
+    "keyboard_folder": "plaid_pad",
+    "url": "https://keycapsss.com/diy-kits/104/plaid-pad-4x4-numpad",
+    "maintainer": "BenRoe",
+    "width": 4,
+    "height": 4,
+    "layouts": {
+        "LAYOUT_numpad_4x4": {
```

Comment:
```suggestion
        "LAYOUT_ortho_4x4": {
```

Reply:
Hi,
can you explain why i should use LAYOUT_ortho_4x4?
It thought it's main purpose is a numpad and then i should use LAYOUT_numpad_4x4.
Sry if this is newbie question.

Reply:
`LAYOUT_numpad_4x4`, as you have it defined in `plaid_pad.h`, is what we call a *layout macro* in QMK. Layout macros are a translation layer between where the keys are physically on the assembled hardware, and where the keys are electrically when the matrix is being scanned.

The `LAYOUT_numpad_4x4` macro is a bit special, because it's one of the [Community Layouts](https://docs.qmk.fm/#/feature_layouts) we have defined. That feature allows users to write one keymap that can be flashed to multiple keyboards, if all the keyboards share the same physical layout (like owning a GH60 Satan, a 1Up 60 RGB, and a Plain 60).

Because it's one of the Community Layouts, we want keyboards that conform to the defined physical layout to use the Community Layout name. Your keyboard here is a 4x4 grid, which conforms to the `ortho_4x4` layout. `numpad_4x4` is a different layout, that has three 2u keys. Observe the difference on the RoadKit by TheVan Keyboards, which supports both layouts:

- ortho_4x4
   ![image](https://user-images.githubusercontent.com/18669334/72381134-b1f36200-36cb-11ea-8b3a-8c6db404043c.png)
- numpad_4x4
   ![image](https://user-images.githubusercontent.com/18669334/72381116-a607a000-36cb-11ea-8edc-84bba1a04738.png)

Just because the designer's concept is to use the device as a numpad, doesn't mean that every user will use it as such.

Reply:
Great explanation. Will change it.